stream.go

```diff
@@ -0,0 +1,150 @@
+package tarantool
+
+import (
+	"fmt"
+	"time"
+
+	"gopkg.in/vmihailenco/msgpack.v2"
+)
+
+type TxnIsolationLevel uint64
+
+const (
+	// By default, the isolation level of Tarantool is serializable.
+	defaultIsolationLevel = iota
```

Comment:
I think it's better to name it `SerializableLevel` and then set up some `DefaultIsolationLevel = SerializableLevel`

Reply:
I think it should be named according to [isolation levels in box protocol](https://www.tarantool.io/en/doc/latest/dev_guide/internals/box_protocol/#iproto-begin-0x0e).

Reply:
Oh, I see, I was wrong in my assumption. Yes, it should be named consistent with core Tarantool.